package.json

```diff
@@ -41,7 +43,9 @@
     "xhr2": "0.1.3"
   },
   "devDependencies": {
+    "@peculiar/webcrypto": "^1.0.9",
```

Comment:
There is a callout in this repository that explicitly states [**not to use this for production**](https://github.com/PeculiarVentures/webcrypto#warning). Did security sign-off on this dependency?

Further, please include our security team on this PR for review.

Reply:
This library is not used in production. it is a devDependency and used only in jest tests where we don't have webcrypto. Jest tests run in a node environment using a jsdom environment. In production we use webcrypto which is built into the browsers.

Reply:
The node `crypto` and `webcrypto` do not have the same interface. I believe this module is using node `crypto` behind the scenes and making it match the interface for `webcrypto`.  

Here is some of the conversation on that: https://github.com/nodejs/node/issues/2833

Another option is to run these tests (involving crypto) in Karma / webpack then they would use the actual webcrypto in the browser. This might be better since it will match what our customers are using.

Reply:
+1 to running the PKCE tests in a browser context.

Reply:
Added karma to run tests in browser using webpack. Removed this dependency.